base/ltsockets.dtx

```diff
@@ -919,10 +920,10 @@
 %    arguments (socket inputs plus one) in the L3 layer code.
 %
 %    \begin{macrocode}
-\cs_new_protected:Npn \socket_use:nw #1 {
-  \@@_debug_term:n
-    { Socket~ '#1'~ containing~ plug~
-      '\str_use:c { l_@@_#1_plug_str }'~ used. }
+\cs_new:Npn \socket_use:nw #1 {
+%  \@@_debug_term:n
+%    { Socket~ '#1'~ containing~ plug~
+%      '\str_use:c { l_@@_#1_plug_str }'~ used. }
```

Comment:
we should probably say why debugging can't be used. In fact we might want to consider supporting a special
\UseAlignSocket as I did had one time --- something to ponder (and perhaps make a remark here).